helm_deploy/values-preprod.yaml

```diff
@@ -6,6 +6,7 @@ generic-service:
 
   ingress:
     host: calculate-release-dates-preprod.hmpps.service.justice.gov.uk
+    contextColour: green
```

Comment:
would be better to set the default as `green` in `values.yaml` and then override for dev, but guess it depends when you're going to production and what cluster that will be in...

Reply:
I think that makes sense. So the next step will be to create the prod set up and that will also be on the live cluster. Then after that dev will be migrated